sdks/java/core/src/main/java/org/apache/beam/sdk/io/TextIO.java

```diff
@@ -649,22 +672,39 @@ public Bound withFooter(@Nullable String footer) {
       public Bound withWritableByteChannelFactory(
           WritableByteChannelFactory writableByteChannelFactory) {
         return new Bound(name, filenamePrefix, filenameSuffix, header, footer, numShards,
-            shardTemplate, validate, writableByteChannelFactory);
+            shardTemplate, validate, writableByteChannelFactory, filenamePolicy, windowedWrites);
+      }
+
+      public Bound withWindowedWrites() {
+        return new Bound(name, filenamePrefix, filenameSuffix, header, footer, numShards,
+            shardTemplate, validate, writableByteChannelFactory, filenamePolicy, true);
```

Comment:
Right now no, you must specify a Strategy. If we want default templates for this, I think that's a valid feature request that should go into a different PR